src/gui/DialogSerialSettings.hpp

```diff
@@ -18,8 +18,7 @@ class DialogSerialSettings : public QDialog
     ~DialogSerialSettings() override;
 
   private slots:
-    void on_serialPortOverrideGroupBox_clicked(); // FIXME: do not use automatic
-                                                  // slot generation
+    void serialPortOverrideBox();
```

Comment:
If you change the slot name, you **must** add the signal/slot connection because it is not done by Qt anymore.